internal/chartverifier/checks/charttesting.go

```diff
@@ -123,7 +123,7 @@ func ChartTesting(opts *CheckOptions) (Result, error) {
 		return NewResult(false, err.Error()), nil
 	}
 
-	_, path, err := LoadChartFromURI(opts.URI)
+	_, path, err := LoadChartFromURI(opts.URI, opts.HelmEnvSettings.RepositoryCache)
```

Comment:
Consider changing to just using opts as the parameter and have LoadChartFromURI pick the values it wants.

Reply:
Yeah makes sense. Made the change.